DataDefinitions/Ship.cs

```diff
@@ -81,6 +79,24 @@ public string name
             }
         }
 
+        private string _model;
```

Comment:
Would prefer future me to know what "_model" actually means and how it differs from the public "model". This name is unclear.